src/sass/components/_advantages.scss

```diff
@@ -1,3 +1,110 @@
 .advantages {
-  background-color: pink;
+  min-width: 320px;
+  background-color: $pageBg;
+  &__list {
+    padding: 39px 20px 59px 20px;
+  }
+  &__header-hidden {
+    display: none;
+  }
+  &__list-header {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    padding: 0 0 38px 0;
+    font-family: 'Titan One';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 36px;
+    line-height: 1.14;
+    letter-spacing: 0.04em;
+    text-transform: uppercase;
+    color: $brand-red;
+    &--1::before {
+      @extend %advantages-icon;
+      background-image: url(../images/icon-1-33.png);
+    }
+    &--2::before {
+      @extend %advantages-icon;
+      background-image: url(../images/icon-2-33.png);
+    }
+    &--3::before {
+      @extend %advantages-icon;
+      background-image: url(../images/icon-3-33.png);
+    }
+  }
+  &__list-text {
+    font-family: 'DM Sans';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 170%;
+    letter-spacing: normal;
+    text-transform: none;
+    color: $body;
+    max-width: 230px;
+  }
+}
+
+%advantages-icon {
+  content: '';
+  height: 50px;
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+@media screen and (min-width: 768px) {
+  .advantages {
+    min-width: 768px;
+    flex-direction: row;
+    &__list {
+      padding: 66px 36px 140px 36px;
+      display: flex;
+    }
+    &__list-header {
+      font-size: 48px;
+      text-transform: none;
+      width: 180px;
+      margin-right: 60px;
+    }
+    &__list-text {
+      font-size: 14px;
+      line-height: 176%;
+    }
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .advantages {
+    min-width: 1280px;
+    &__list {
+      padding: 39px 115px 140px 115px;
```

Comment:
container załatwi sprawę paddingów lewy i prawy, możesz je stąd usunąć. A jeżeli chodzi o górny i dolny to najlepiej użyć go na section